content/github/setting-up-and-managing-your-enterprise/roles-in-an-enterprise.md

```diff
@@ -31,7 +31,7 @@ Enterprise owners have complete control over the enterprise and can take every a
 
 Enterprise owners cannot access organization settings or content unless they are made an organization owner or given direct access to an organization-owned repository. Similarly, owners of organizations in your enterprise do not have access to the enterprise itself unless you make them enterprise owners.
 
-You can add as many enterprise owners as you'd like to your enterprise. {% if currentVersion == "free-pro-team@latest" %}Enterprise owners must have a personal account on {% data variables.product.prodname_dotcom %}.{% endif %} As a best practice, we recommend making only a few people in your company enterprise owners, to reduce the risk to your business.
+You can add as many enterprise owners as you'd like to your enterprise as they do not consume a license unless they are an owner or member of at least one organization within the enterprise. {% if currentVersion == "free-pro-team@latest" %}Enterprise owners must have a personal account on {% data variables.product.prodname_dotcom %}.{% endif %} As a best practice, we recommend making only a few people in your company enterprise owners, to reduce the risk to your business.
```

Comment:
I actually think your rewording is fine 😄  If you wanted it to be a bit more concise you can also do this:

```suggestion
An enterprise owner will only consume a license if they are an owner or member of at least one organization within the enterprise. {% if currentVersion == "free-pro-team@latest" %}Enterprise owners must have a personal account on {% data variables.product.prodname_dotcom %}.{% endif %} As a best practice, we recommend making only a few people in your company enterprise owners, to reduce the risk to your business.
```

There's a section for billing managers down below as well that we can update with similar wording (but I'm guessing this is also true for billing managers, yeah?)

Reply:
Will adopt the same wording for billing managers! Thanks @megbird